connaisseur/workload_object.py

```diff
@@ -48,7 +50,7 @@ def __init__(self, request_object: dict, namespace: str):
                 wl_obj_name=self.name,
             )
 
-    @property
+    @cached_property
```

Comment:
Out of curiosity, how much does that save?

Reply:
And nit: should be its own commit to improve performance of other feature

Reply:
haven't done a test, but previously, if there where multiple containers defined, conny would have made a network request for the parent for each container ... so for the standard case barly any changes, but for handcraft one substantial i'd assume.

so i added it for each "expensive" property.

Reply:
will do another commit after review + rebasing